test/lib/SDL.jl

```diff
@@ -23,7 +23,11 @@ win = SDL2.CreateWindow("Hello World!", Int32(100), Int32(100), Int32(300), Int3
 
 renderer = SDL2.CreateRenderer(win, Int32(-1),
              UInt32(SDL2.RENDERER_ACCELERATED | SDL2.RENDERER_PRESENTVSYNC))
-@test renderer != C_NULL
+if get(ENV, "HAS_JOSH_K_SEAL_OF_APPROVAL", "") == "true" && Sys.islinux()
```

Comment:
What are those about ?

Reply:
Well, we couldn't find a way to make tests pass on Linux with Travis, `xvfb-run` doesn't seem to help here. They do pass locally for me though. This is marking the tests as broken only on Linux in Travis

Reply:
I see that Linux on Travis never actually worked so far? I think this is still an improvement. Note that these tests are expected to pass on macOS and Windows with Travis and AppVeyor.

Reply:
Yes, these line disable a few tests when running on linux on Travis. The `HAS_JOSH_K..` env setting is a documented way to test for travis. When running locally on linux, these tests are still executed. The previous setting had the linux tests completey disregarded on linux, so I think this is an improvement (even if not ideal)

Reply:
Alright, thanks for the explanation.